src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6052,4 +6052,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_GeneratorFailedDuringInitialization_Title" xml:space="preserve">
     <value>Generator failed to initialize.</value>
   </data>
+  <data name="IDS_FeatureExtensionGetAsyncEnumerator" xml:space="preserve">
+    <value>extension GetAsyncEnumerator</value>
+  </data>
+  <data name="IDS_FeatureExtensionGetEnumerator" xml:space="preserve">
+    <value>extension GetEnumerator</value>
+  </data>
+  <data name="ERR_PatternStaticOrInaccessible" xml:space="preserve">
+    <value>'{0}' does not implement the '{1}' pattern. '{2}' is either static or not public.</value>
```

Comment:
Can't say I'm fan of the "is either static or not public" bit. In one pattern case you want it to be an instance method, and in the other you want it to be static. Consider breaking these up, or being more specific (something like `'{2} is not an accessible instance or extension method`).